app/admin/governance/multitenant_wallet/wallet.py

```diff
@@ -12,10 +12,10 @@
 
 logger = logging.getLogger(__name__)
 
-router = APIRouter(prefix="/wallets", tags=["wallets"])
+router = APIRouter(prefix="/wallets", tags=["Admin: Wallets"])
```

Comment:
Should this not be /admin/wallets perhaps

Reply:
Thanks for your input there @wip-abramson - I do agree. Though it's not part of this PR's purview.